README.md

```diff
@@ -310,6 +310,20 @@ Or install it yourself as:
   ticker.last
 ```
 
+### Query the Ticker stream API
+
+```
+  pair = client.pairs('bitfinex').first
+  thread = client.ticker_stream(
+    market_pair: pair,
+    onopen: Proc.new { puts 'Opened!' },
+    onmessage: Proc.new { |ticker| puts do_something_with_ticker(ticker) },
```

Comment:
I would put a working example here to print the ticker value rather than `do_something_with_ticker`

In fact when `do_something_with_ticker` was used, there was no indication of any error thrown. No mention of undefined method etc.